src/main/java/picard/util/LiftoverUtils.java

```diff
@@ -102,14 +102,18 @@ public static VariantContext liftVariant(final VariantContext source, final Inte
         }
 
         if (writeOriginalAlleles && !source.getAlleles().equals(builder.getAlleles())) {
-            final List<String> alleles = new ArrayList<>();
-            source.getAlleles().forEach(a -> alleles.add(a.getDisplayString()));
-            builder.attribute(LiftoverVcf.ORIGINAL_ALLELES, alleles);
+            builder.attribute(LiftoverVcf.ORIGINAL_ALLELES, allelesToStringList(source.getAlleles()));
         }
 
         return builder.make();
     }
 
+    protected static List<String> allelesToStringList(final List<Allele> alleles) {
```

Comment:
could you add a few small independent tests to this new static function? Also, despite it being non-public, could you add a javadoc so it's clear what this function intends to do?

Reply:
done